### PR TITLE
Objass

### PR DESCRIPTION
**Pull request by pguzman26**
Please review and let me know if I fixed it correctly(I think I did) :)

### CODE REVIEW

**Comment by pguzman26** [comment]
Please review. Thanks

File: lib/assessment.rb [hunk 1]
```diff
@@ -6,13 +6,25 @@
 # Make all of those attributes read-only.
 
 ##
-# your code here
+class Dog
+  attr_reader :name
+  attr_reader :breed
+  attr_reader :age
```
Comment (jrhorn424) [inline lib/assessment.rb:12]: Good! This can be done in one line, though. `attr_reader :attr1, :attr2, :etc`

File: lib/assessment.rb [hunk 2]
```diff
@@ -20,20 +32,32 @@
 # Name and location should be write-only, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_accessor :name
+  attr_reader :age
+  attr_accessor :location
```
Comment (jrhorn424) [inline lib/assessment.rb:38]: This is not write-only. `attr_accessor` does both read and write (setter and getter).

**Comment by jrhorn424** [comment]
4.5/5 Good work! How is your comfort with this material?

**Comment by pguzman26** [comment]
I am very comfortable with this.

**Pull request closed without merging**